doc/connectors/google_sheets_2.md

```diff
@@ -14,6 +15,8 @@ The `baseroute` is fixed and is 'https://sheets.googleapis.com/v4/spreadsheets/'
 
 The `secrets` dictionary contains the `access_token` and a `refresh_token` (if there is one). Though `secrets` is optional during the initial creation of the connector, it is necessary for when the user wants to make requests to the connector. If there is no `access_token`, an Exception is thrown.
 
+The `auth_flow_id` property is like a identifier that is used to identify the connector.
```

Comment:
```suggestion
The `auth_flow_id` property is like a identifier that is used to identify the secrets associated to the connector.
```

Reply:
Yes j'ai aussi fait une bêtise ... ça devrait être 'is like an identifier'... ne dis pas aux autres :blush: